src/analysis/vector/geometry_checker/qgsgeometrycheck.cpp

```diff
@@ -56,6 +56,27 @@ void QgsGeometryCheck::fixError( const QMap<QString, QgsFeaturePool *> &featureP
   Q_UNUSED( changes )
 }
 
+QList<QgsGeometryCheckResolutionMethod> QgsGeometryCheck::availableResolutionMethods() const
+{
+  QList<QgsGeometryCheckResolutionMethod> fixes;
+  Q_NOWARN_DEPRECATED_PUSH
```

Comment:
Can you add a comment about how this function should be fixed when resolutionMethods() is removed?